### DISCUSSION TITLE
Installer thinks that php modules are missing.

### DISCUSSION THREAD

**Discussion by Bravesun** [firefly-iii/firefly-iii#4762] [Firefly III]
I am trying to install Firefly III version 5.5.11.

**Description**
I get this error while trying to install/upgrade 5.5.11, PHP version is 7.4.16
```
Problem 1
    - The requested PHP extension ext-intl * is missing from your system. Install or enable PHP's intl extension.
```

I've used phpinfo() to confirm that all the required modules are installed and active but the installation script still thinks that intl is missing.

**Comment by JC5** [comment]
Could it be that it's not enabled for the php cli?
